frontend/my-app/src/App.js

```diff
@@ -1,11 +1,36 @@
 import './App.css';
 import CreateMeeting from './components/CreateMeeting';
+import LineChartDashboard from './components/LineChartDashboard';
+import BarChartHorizontalDashboard from './components/BarChartHorizontalDashboard';
+import GenMeetingBarChart from './components/GenMeetingBarChart';
+import GenMeetingDonutChart from './components/GenMeetingDonutChart';
+import MemberTable from './components/MemberTable';
+
 function App() {
   return (
     <div className="App">
       <CreateMeeting>
-
       </CreateMeeting>
+
+      <div style={{ width: 700 }}>
```

Comment:
You seem to use width: 700 alot leading me to think that you want all the fields to be that width, maybe make one div with that width:700 style and use it for all of these wrapper divs